src/v2/cookbook/creating-custom-scroll-directives.md

```diff
@@ -96,6 +96,57 @@ new Vue({
 
 Though we would remove the previous CSS transition from this implementation because it's now handled with JavaScript.
 
+## Another Example
```

Comment:
Call me conservative, but I don't find the need for another example here either.